pyrk/materials/fuel.py

```diff
@@ -0,0 +1,44 @@
+from ur import units
+from material import Material
+from density_model import DensityModel
+import random
+
+class Fuel(Material):
```

Comment:
I have the same concern about this material name as I had about Coolant. What kind of fuel are you describing? UOXKernels?